### ISSUE TITLE
npm install pizzip fails on windows

### ISSUE THREAD

**Issue by jesuslpm** [open-xml-templating/pizzip#2]
npm install pizzip fails on windows with 'rm' is not recognized as an internal or external command,

```
'rm' is not recognized as an internal or external command,
operable program or batch file.
npm WARN enoent ENOENT: no such file or directory, open 'D:\caca\package.json'
npm WARN caca No description
npm WARN caca No repository field.
npm WARN caca No README data
npm WARN caca No license field.
npm ERR! Windows_NT 10.0.17134
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\jesus.lopez.IN2007\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "pizzip"
npm ERR! node v10.15.3
npm ERR! npm  v4.0.2
npm ERR! code ELIFECYCLE

npm ERR! pizzip@3.0.5 preinstall: `rm -rf utils/node_modules`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the pizzip@3.0.5 preinstall script 'rm -rf utils/node_modules'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the pizzip package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm -rf utils/node_modules
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs pizzip
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls pizzip
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\caca\npm-debug.log
```

**Comment by boriskurikhin** [comment]
Try using PowerShell?

**Comment by edi9999** [comment]
This has been fixed in version 3.0.6
